experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeCodecs.ts

```diff
@@ -46,7 +44,7 @@ import {
 	EncodedRevisionInfo,
 } from "./modularChangeFormat";
 
-function makeV0Codec(
+export function makeModularChangeCodec(
```

Comment:
What is the motivation to remove codec versioning for this?

Reply:
Discussed offline. Reverted!